apiserver/storage/storage.go

```diff
@@ -190,6 +191,7 @@ func createStorageDetails(st storageAccess, si state.StorageInstance) (*params.S
 				a.Unit().String(),
 				machineTag.String(),
 				location,
+				params.Life(a.Life().String()),
```

Comment:
would prefer named fields
we use named fields below in StorageDetails struct initialisation